config/config.go

```diff
@@ -151,3 +151,14 @@ func AddFlags(cmd *cobra.Command) {
 	cmd.Flags().String(FlagTrustedHash, def.TrustedHash, "initial trusted hash to start the header exchange service")
 	cmd.Flags().Uint64(FlagMaxPendingBlocks, def.MaxPendingBlocks, "limit of blocks pending DA submission (0 for no limit)")
 }
+
+type TomlConfig struct {
```

Comment:
nit: I would put this with the `ReadToml` code in a file call `toml.go` 
Then it is more explicitly clear what the code is for.